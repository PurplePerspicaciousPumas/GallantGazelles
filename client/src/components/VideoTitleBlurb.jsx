import React from 'react';
import { Segment } from 'semantic-ui-react';

export default (props) => (
  <section>
    <div>
      <p>Shift</p>
      <p>Switch between Gmail accounts like a boss</p>
    </div>
  </section>
)

export default VideoTitleBlurb;