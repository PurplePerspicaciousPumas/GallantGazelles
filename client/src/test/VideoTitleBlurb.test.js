import React from 'react';
import VideoTitleBlurb from '../components/VideoTitleBlurb.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <VideoTitleBlurb />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});