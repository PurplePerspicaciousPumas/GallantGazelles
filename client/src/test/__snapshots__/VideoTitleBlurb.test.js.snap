// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div>
    <p>
      Shift
    </p>
    <p>
      Switch between Gmail accounts like a boss
    </p>
  </div>
</section>
`;
